proto/master.proto

```diff
@@ -120,3 +124,17 @@ message ScheduleResult {
 message TaskSchedulerResponse {
     map<int64, ScheduleResult> schedule = 1;
 }
+
+message ExecWorkload {
+    JobType tp = 1;
```

Comment:
maybe more content should be reported to the resource manager

Reply:
yes, we can add when we need more fields.